File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;// Defines height

    // Loop to get acceptable height
    while (height < 1 || height > 8)
    {
        height = get_int("Height: ");
    }

    int spaces = height - 1; // Counter for spaces to print

    // main for loop for passing through levels
    for (int i = 1; i <= height; i++)
    {
        for (int j = 1; j <= spaces; j++)
        {
            printf(" ");
        }
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n"); // breakline
        spaces--; // spaces counter update
    }
}

File: sentimental-mario-more/mario.py
# Replicates mario.c

from cs50 import get_int


def main():
    height = 0
    while height < 1 or height > 8:
        height = get_int("Height: ")
    # Counter for spaces to print
    spaces = height - 1
    # main for-loop for passing through levels
    for i in range(1, height + 1):
        for j in range(1, spaces + 1):
            print(" ", end='')
        for j in range(1, i + 1):
            print("#", end='')
        print("  ", end='')
        for j in range(1, i + 1):
            print("#", end='')
        # breakline
        print("\n", end='')
        # spaces counter update
        spaces -= 1


if __name__ == "__main__":
    main()